slack-clone-compose-sample/src/main/kotlin/io/getstream/slack/compose/ui/theme/Typography.kt

```diff
@@ -0,0 +1,55 @@
+package io.getstream.slack.compose.ui.theme
+
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import io.getstream.chat.android.compose.ui.theme.StreamTypography
+import io.getstream.slack.compose.R
+
+private val light = Font(R.font.lato_light, FontWeight.W300)
+private val regular = Font(R.font.lato_regular, FontWeight.W400)
+private val medium = Font(R.font.lato_regular, FontWeight.W500)
+private val bold = Font(R.font.lato_bold, FontWeight.W700)
+
+private val slackFontFamily = FontFamily(fonts = listOf(light, regular, medium, bold))
+
+/**
+ * The default [StreamTypography] from the SDK with a custom font family
+ * used by Slack ([Lato](https://fonts.google.com/specimen/Lato)).
+ */
+val slackTypography: StreamTypography = StreamTypography
+    .defaultTypography()
+    .withFontFamily(slackFontFamily)
+
+/**
+ * Returns a copy of [StreamTypography] with a custom font family applied to
+ * every text style..
+ *
+ * @param fontFamily The font family to be used with every text style.
+ */
+private fun StreamTypography.withFontFamily(fontFamily: FontFamily): StreamTypography {
```

Comment:
This is a good point. We're currently missing the font family customization, so we'll have to add this too!